protobufs/entity/service.proto

```diff
@@ -94,3 +94,22 @@ message EntityData {
   // Entity contact list
   repeated Contact contacts = 5;
 }
+
+message EntityRelationshipData {
+  string owner_identity = 1;
+  string label = 2;
```

Comment:
I am wondering if we need this label at all here, as we will add more info with custom attributes in the near future.